lib/fog/libvirt/models/compute/server.rb

```diff
@@ -351,14 +351,18 @@ def defaults
           }
         end
 
+        def valid_boot_media
+          %w[cdrom fd hd network]
+        end
+
         def default_boot_order
           %w[hd cdrom network]
         end
 
         def verify_boot_order order = []
           if order
             order.each do |b|
-              raise "invalid boot order, possible values are: hd, network and/or cdrom" unless default_boot_order.include?(b)
```

Comment:
`:boot_order` is initialized to `default_boot_order` in the `defaults` method just a few lines up, so I guess it's still relevant. Admittedly, I have not looked at the full context of how any of this is used -- I only debugged this very locally/contextless.